### PR TITLE
Log messages at debug level

### PR DESCRIPTION
**Pull request by houcros**

### CODE REVIEW

File: libats-logging/src/main/resources/logback-libats.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <logger name="com.zaxxer.hikari" level="${HIKARI_LOG_LEVEL:-INHERITED}" />
     <logger name="slick.backend.DatabaseComponent.action" level="${SLICK_LOG_LEVEL:-INHERITED}" />
     <logger name="com.advancedtelematic" level="${ATS_LOG_LEVEL:-INHERITED}" />
-    <logger name="com.advancedtelematic.libats.messaging.kafka" level="${ATS_KAFKA_LOG_LEVEL:-INHERITED}" />
+    <logger name="com.advancedtelematic.libats.messaging" level="${ATS_MESSAGING_LOG_LEVEL:-INHERITED}" />
```
Comment (houcros) [inline libats-logging/src/main/resources/logback-libats.xml]: There's more logging going on at this level than in the `kafka` subpackage, so we can handle all of it from here. Alternatively I could create another logger for the subpackage but that doesn't seem worth it.

Comment (simao) [inline libats-logging/src/main/resources/logback-libats.xml]: Why this change? `.kafka` is there because of https://github.com/advancedtelematic/libats/blob/7e461829fca65a11bccc4d84f1947fa6351995b3/libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/kafka/KafkaClient.scala#L85

Comment (houcros) [reply libats-logging/src/main/resources/logback-libats.xml]: Am I wrong assuming that this will handle logs under both `com.advancedtelematic.libats.messaging` and `com.advancedtelematic.libats.messaging.kafka` ?

File: libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/daemon/MessageBusListenerActor.scala [hunk 2]
```diff
@@ -83,7 +83,7 @@ object MessageBusListenerActor {
   private lazy val log = LoggerFactory.getLogger(this.getClass)
 
   def loggingSink[M](implicit ml: MessageLike[M]): Sink[M, Future[Done]] = Sink.foreach[M] { msg =>
-    log.info(s"Processed ${ml.streamName} - ${ml.id(msg)}")
+    log.debug(s"Processed ${ml.streamName} - ${ml.id(msg)}")
```
Comment (houcros) [inline libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/daemon/MessageBusListenerActor.scala]: I think this might be a better default? Else we're gonna fill Splunk with a log of each event...

File: libats-logging/src/main/resources/logback-libats.xml (hunk 1 above)
Comment (simao) [reply libats-logging/src/main/resources/logback-libats.xml]: Yeah but I mean why not have `ATS_KAFKA_LOG_LEVEL` anymore?

Comment (houcros) [reply libats-logging/src/main/resources/logback-libats.xml]: Ah so you'd rather have one logger for each. I just thought we didn't wouldn't be interested in that level of control, but yes sure, will update.

Comment (simao) [reply libats-logging/src/main/resources/logback-libats.xml]: I don't have a preference but I wanted to know why the change

Comment (houcros) [reply libats-logging/src/main/resources/logback-libats.xml]: If you mean the change in the variable name it's just for consistency with the package. I did a `ack ATS_KAFKA_LOG_LEVEL` at `ota-charts` and found no usage so I figured changing the variable name wouldn't have any implications.

Comment (simao) [reply libats-logging/src/main/resources/logback-libats.xml]: Ah! Ok I thought ATS_MESSAGING_* already existed. ok ok
